keyboards/mokey/mokey64/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = STM32F401
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = full     # Virtual DIP switch configuration
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration
```

Reply:
It has been modified based on the suggestions made